packages/openapi-typescript/src/transform/paths-object.ts

```diff
@@ -26,8 +27,9 @@ const PATH_PARAM_RE = /\{[^}]+\}/g;
 export default function transformPathsObject(
   pathsObject: PathsObject,
   ctx: GlobalContext,
-): ts.TypeNode {
+): [ts.TypeNode, ts.TypeAliasDeclaration[]] {
```

Comment:
```suggestion
): { node: ts.TypeNode, aliases: ts.TypeAliasDeclaration[] } {
```

This is a personal opinion, but let’s not return tuple types for any functions; those don’t scale well. If a function needs to return multiple things, a **named object** is much better (same for parameters—named objects are more maintainable than ordered params or tuples)